packages/x-data-grid/src/components/virtualization/GridVirtualScrollerRenderZone.tsx

```diff
@@ -33,20 +32,14 @@ const GridVirtualScrollerRenderZone = forwardRef<
   HTMLDivElement,
   React.HTMLAttributes<HTMLDivElement> & { sx?: SxProps<Theme> }
 >(function GridVirtualScrollerRenderZone(props, ref) {
-  const { className, ...other } = props;
-  const apiRef = useGridPrivateApiContext();
   const rootProps = useGridRootProps();
   const classes = useUtilityClasses(rootProps);
-  const offsetTop = apiRef.current.virtualizer.api.getters.getOffsetTop();
 
   return (
     <VirtualScrollerRenderZoneRoot
-      className={clsx(classes.root, className)}
       ownerState={rootProps}
-      style={{
-        transform: `translate3d(0, ${offsetTop}px, 0)`,
-      }}
```

Comment:
I've moved all responsability to generate styles to a single place, `layout.ts` in the virtualizer package.